drivers/src/nrfx_spim.c

```diff
@@ -263,6 +263,10 @@ nrfx_err_t nrfx_spim_init(nrfx_spim_t  const * const p_instance,
     }
     m_cb[p_instance->drv_inst_idx].miso_pin = p_config->miso_pin;
     // - Slave Select (optional) - output with initial value 1 (inactive).
+	
+    // Any slave select configuration has to be saved for conditions in spim_xfer function.
+    m_cb[p_instance->drv_inst_idx].ss_pin = p_config->ss_pin;
```

Comment:
I would replace `m_cb[p_instance->drv_inst_idx].` with `p_cb->` (for other pins as well) and change the comment to something like:
'p_cb->ss_pin' is used during transfers to check if SS pin should be toggled, so this field needs to be initialized even if the pin is not used.